src/SuperVault.sol

```diff
@@ -58,6 +61,14 @@ contract SuperVault is BaseStrategy, ISuperVault {
         _;
     }
 
+    /// @notice Ensures that only the Vault Manager can call the function
+    modifier onlyVaultManager() {
+        if (_getAddress(keccak256("VAULT_MANAGER")) != msg.sender) {
```

Comment:
How are we retrieving this address? Notice "management" is a variable within TokenizedStrategy.sol which this contract can access to via _strategyStorage() call